packages/invoices/src/OutboundShipment/DetailView/modals/BatchesTable.tsx

```diff
@@ -67,37 +35,22 @@ const BatchesRow: React.FC<BatchesRowProps> = ({
   disabled,
   onChange,
 }) => {
-  const { register } = useFormContext();
-  const t = useTranslation('common');
   const d = useFormatDate();
 
-  const onChangeValue: React.ChangeEventHandler<HTMLInputElement> = event => {
-    const value = Math.max(Number(event.target.value), 0);
-    const newValue = Math.min(value, batch.availableNumberOfPacks);
-
-    onChange?.(batch.id, newValue, batch.packSize);
-  };
-
-  const stockLineInputProps = register(batch.id, {
-    min: { value: 0, message: t('error.invalid-value') },
-    max: {
-      value: batch.availableNumberOfPacks,
-      message: t('error.invalid-value'),
-    },
-    pattern: { value: /^[0-9]+$/, message: t('error.invalid-value') },
-    onChange: onChangeValue,
-  });
```

Comment:
The `NumericTextInput` handles a bunch of this but not the error messages - but I thought at this point we can maybe handle that a different way? 🤷

Reply:
the input should prevent most of the invalid values anyway, so perhaps the error messaging isn't required